src/main/java/frc/team555/robot/PowerUpRobot.java

```diff
@@ -0,0 +1,75 @@
+package frc.team555.robot;
+
+import edu.wpi.first.wpilibj.interfaces.Gyro;
+import org.montclairrobotics.sprocket.SprocketRobot;
+import org.montclairrobotics.sprocket.control.ButtonAction;
+import org.montclairrobotics.sprocket.control.SquaredDriveInput;
+import org.montclairrobotics.sprocket.drive.*;
+import org.montclairrobotics.sprocket.drive.steps.Deadzone;
+import org.montclairrobotics.sprocket.drive.steps.GyroCorrection;
+import org.montclairrobotics.sprocket.drive.utils.GyroLock;
+import org.montclairrobotics.sprocket.geometry.Vector;
+import org.montclairrobotics.sprocket.geometry.XY;
+import org.montclairrobotics.sprocket.motors.Module;
+import org.montclairrobotics.sprocket.motors.Motor;
+import org.montclairrobotics.sprocket.pipeline.Pipeline;
+import org.montclairrobotics.sprocket.pipeline.Step;
+import org.montclairrobotics.sprocket.utils.Input;
+import org.montclairrobotics.sprocket.utils.PID;
+
+import java.util.ArrayList;
+
+public class PowerUpRobot extends SprocketRobot {
+
+    Hardware hardware;
+    DriveTrain driveTrain;
+    Gyro navx;
+    GyroCorrection correction;
+    GyroLock lock;
+
+    @Override
+    public void robotInit(){
+        Hardware.init();
+        DriveModule[] modules = new DriveModule[4];
+        modules[0] = new DriveModule(new XY(-1, -1), Vector.ZERO, new Motor(Hardware.motorDriveBL));
+        modules[1] = new DriveModule(new XY(1, -1), Vector.ZERO, new Motor(Hardware.motorDriveBR));
+        modules[2] = new DriveModule(new XY(-1, 1), Vector.ZERO, new Motor(Hardware.motorDriveFL));
+        modules[3] = new DriveModule(new XY(1, 1), Vector.ZERO, new Motor(Hardware.motorDriveFL));
+
+        driveTrain = new DriveTrain(modules);
```

Comment:
Be sure you're instantiating properly with the constructor (I'm not sure if it's correct or not, will require testing).

Reply:
Okay, ill test when we have the robot